examples/favorites_advanced/getx_favorites_advanced/lib/base/ui_components/puppies_app_bar.dart

```diff
@@ -41,6 +41,16 @@ class PuppiesAppBar extends StatelessWidget implements PreferredSizeWidget {
     }
   }
 
+  Widget _createTitle() =>
+      controller.filteredBy.value != ''
```

Comment:
@RomanovaPrime, here you could use [isNotEmpty](https://api.dart.dev/stable/2.3.2/dart-core/String/isNotEmpty.html) instead `!= ''`